R/methods_kmeans.R

```diff
@@ -123,7 +123,12 @@ print.parameters_clusters <- function(x, digits = 2, ...) {
 # Predict -----------------------------------------------------------------
 
 
+#' Predict method for parameters_clusters objects
```

Comment:
I think this is the reason why pkgdown fails, because `predict.parameters_clusters()` is now indexed in package-down. You need to add an entry `_pkgdown.yaml` as well, if you create a help page for this method.